src/PKSim.Presentation/Presenters/Individuals/IndividualSettingsPresenter.cs

```diff
@@ -116,7 +119,8 @@ private IReadOnlyList<DiseaseState> allDiseaseStatesFor(SpeciesPopulation popula
 
       public void PrepareForCreating()
       {
-         var individual = _defaultIndividualRetriever.DefaultIndividual();
+         //clone individual to ensure that we do not modify the default individual cached
+         var individual = _cloner.Clone(_defaultIndividualRetriever.DefaultIndividual());
```

Comment:
cloning the individual here feels like the best way to avoid any side effects. We probably have been dealing with this bug for quite a while